src/components/shared/Dropdown.js

```diff
@@ -47,7 +48,7 @@ class Dropdown extends Component {
   renderButton() {
     return (
       <button className="dropdown-button" onClick={this.toggleDropdown}>
-        »
+        <Svg name="plus" />
```

Comment:
We should be able to pass the button down to `Dropdown`: so that » is used for Showmore